lib/active_model/serializer.rb

```diff
@@ -6,6 +6,8 @@
 require 'active_model/serializer/fieldset'
 require 'active_model/serializer/lint'
 
+# ActiveModel::Serializer is an abstract class that is
+# reified when subclassed to decorate a resource.
```

Comment:
Nope, i mean reified.  Is just fancy for 'made real'. Intention is like instantiated but for object not instance

B mobile phone

> On Oct 12, 2015, at 4:19 AM, Ben A. Morgan <notifications@github.com> wrote:
> 
> In lib/active_model/serializer.rb:
> 
> > @@ -6,6 +6,8 @@
> >  require 'active_model/serializer/fieldset'
> >  require 'active_model/serializer/lint'
> >  
> > +# ActiveModel::Serializer is an abstract class that is
> > +# reified when subclassed to decorate a resource.
> Typo here with reified. Do you mean redefined?
> 
> —
> Reply to this email directly or view it on GitHub.
>